app/renderer/components/tabs/tab.js

```diff
@@ -137,19 +140,26 @@ class Tab extends React.Component {
   onMouseLeave (e) {
     // mouseleave will keep the previewMode
     // as long as the related target is another tab
+    clearTimeout(this.mouseTimeout)
     windowActions.setTabHoverState(this.props.frameKey, false, hasTabAsRelatedTarget(e))
   }
 
   onMouseEnter (e) {
     // if mouse entered a tab we only trigger a new preview
     // if user is in previewMode, which is defined by mouse move
+    clearTimeout(this.mouseTimeout)
     windowActions.setTabHoverState(this.props.frameKey, true, this.props.previewMode)
   }
 
   onMouseMove () {
     // dispatch a message to the store so it can delay
     // and preview the tab based on mouse idle time
-    windowActions.onTabMouseMove(this.props.frameKey)
+    clearTimeout(this.mouseTimeout)
```

Comment:
would `this.onMouseMove = debounce(this.onMouseMove.bind(this), this.mouseTimeout)` not work in the constructor?

Reply:
I am not sure how to clearTimeout with debouce function? If we can use clear then this can be used yeah

Reply:
I think we should not use debounce here. Debouncing leads to bad preview behavior when `previewMode` is enabled after consecutive previews